syntax = "proto3";

/*option java_multiple_files = true;*/
/*option java_package = "io.grpc.examples.helloworld";*/
/*option java_outer_classname = "HelloWorldProto";*/
/*option objc_class_prefix = "HLW";*/

package suresecureivs;

/*import "common.suresecureivs.proto";*/

service ImageAnalysis {
  // Sends a greeting
  rpc ImageClassify (ImageRegion) returns (ImageClassifyReply) {}
}

message ImageRegion{
  bytes img = 1;
  int32 x = 2;
  int32 y = 3;
  int32 w = 4;
  int32 h = 5;
}

message ImageClassifyReply{
  enum Type{
    BACK_GROUND = 0;
    PERSON = 1;
  }
  Type type = 1;
}
